### PR TITLE
Small type hint fixing + added makefile with couple commands.

### PR DESCRIPTION
**Pull request by Abdujabbar**
Hello!

I've found some places where type hinting was missed, so I decided to fix these places and also added Makefile to simplify the command run.

### CODE REVIEW

**Comment by Abdujabbar** [comment]
@microsoft-github-policy-service agree

**Comment by gagb** [comment]
@jackgerrits do you think it makes sense to mix make files and pyproject.toml ?

**Comment by gagb** [comment]
Related #46

File: Makefile
Comment (gagb) [inline Makefile:1]: Is there a way of doing without including a Makefile? E.g., by modifying the pyproject.toml?

**Comment by jackgerrits** [comment]
> @jackgerrits do you think it makes sense to mix make files and pyproject.toml ?

Looks like this repo has no existing scripts or a system for defining scripts to run. A make file is totally reasonable and lightweight :) Can always be replaced if something like poe is used in future

**Review by jackgerrits — APPROVED** [review]
LGTM!

**Comment by gagb** [comment]
@afourney are you okay with including a make file?

**Comment by afourney** [comment]
> @afourney are you okay with including a make file?

Sure. If @jackgerrits says it's fine, I'll take that as gospel :)

**Comment by gagb** [comment]
Sorry @Abdujabbar, we appreciate your contributions though.

**Pull request closed without merging**